src/device/functionality/calibration/update.js

```diff
@@ -8,27 +8,16 @@ const { DEFAULT_REQ_OPTS } = require('../../../utils/defaultReqOpts')
  * @memberof nexus.device.functionality.calibration
  * @function update
  * @async
- * @param {Object} params
- * @param {string} params.deviceId
- * @param {string} params.fctId
- * @param {string} params.slotName
- * @param {Object} params.params
- * @param {Object} params.params.offset
- * @param {Object} params.params.slope
- * @param {number} params.params.offset.value
- * @param {boolean} params.params.offset.locked
- * @param {number} params.params.slope.value
- * @param {boolean} params.params.slope.locked
+ * @param {string} deviceId
+ * @param {string} fctGraphId
+ * @param {string} fctId
+ * @param {string} slotName
+ * @param {Object} payload
```

Comment:
are we calling this `payload` or `body`?

Reply:
I like the approach of using the full body as the parameter instead of building it from the single props. 
But I think we still should keep the docks for expected properties. How would you know otherwise what the expected properties are?

Reply:
I know that is is somewhat inconsistent, as when something changes, you would need to update the docs in the same fashion you would need to change the code as it was before this refactoring, but updating docs could become an operation that will not create a version bump.

Reply:
body won, payload is only used in some tests